docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -115,7 +115,7 @@ After activating the option to use the configuration file:
 
 -   Codacy will use the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results will take those changes into account.
 -   If Codacy analyzes a branch that doesn't include the configuration file, Codacy reverts to using the code patterns configured for the tool before you selected the option **Configuration file** on the Code patterns page.
--   Codacy will use tool configuration files even if you ignore them [using the Codacy UI](ignoring-files.md) or [in a Codacy configuration file](codacy-configuration-file.md).
+-   Codacy will use tool configuration files even if you ignore them for the Codacy analysis [using the Codacy UI](ignoring-files.md) or a [Codacy configuration file](codacy-configuration-file.md).
```

Comment:
The way you word it in this PR's comment, `exclude them from the Codacy analysis`, sounds more natural. Thoughts? 🤔 

```suggestion
-   Codacy will use tool configuration files even if you exclude them from the Codacy analysis [using the Codacy UI](ignoring-files.md) or a [Codacy configuration file](codacy-configuration-file.md).
```

Reply:
Just noticing that we're using the future tense. How about `Codacy keeps using tool configuration files...`?

Reply:
> The way you word it in this PR's comment, `exclude them from the Codacy analysis`, sounds more natural. Thoughts?

I was trying to keep the name of the "Ignore files" feature in there. But I guess it's OK to not include it, since indeed the sentence becomes more straightforward. :+1: 

> Just noticing that we're using the future tense. How about `Codacy keeps using tool configuration files...`?

Here I opted to keep the consistency with the instructions and other sentenced above the line being edited. And I'm guessing that I originally opted for the future tense because the configuration change only takes effect after Codacy performs a new analysis.